actors/market/src/types.rs

```diff
@@ -22,41 +22,40 @@ use super::deal::{ClientDealProposal, DealProposal, DealState};
 pub const PROPOSALS_AMT_BITWIDTH: u32 = 5;
 pub const STATES_AMT_BITWIDTH: u32 = 6;
 
-#[derive(Serialize_tuple, Deserialize_tuple)]
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
 pub struct WithdrawBalanceParams {
     pub provider_or_client: Address,
     pub amount: TokenAmount,
 }
 
 impl Cbor for WithdrawBalanceParams {}
 
-#[derive(Serialize_tuple, Deserialize_tuple)]
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
 #[serde(transparent)]
 pub struct WithdrawBalanceReturn {
     pub amount_withdrawn: TokenAmount,
 }
 
-#[derive(Serialize_tuple, Deserialize_tuple)]
-pub struct OnMinerSectorsTerminateParams {
-    pub epoch: ChainEpoch,
-    pub deal_ids: Vec<DealID>,
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
+pub struct GetBalanceReturn {
+    pub balance: TokenAmount,
+    pub locked: TokenAmount,
```

Comment:
I think it's generally advisable to add
`impl Cbor for GetBalanceReturn {}`

Reply:
It's already inconsistent, and my policy now is to not add it: https://github.com/filecoin-project/builtin-actors/issues/700